### DISCUSSION TITLE
Custom Default Settings

### DISCUSSION THREAD

**Discussion by Tthongsy** [datalust/seq-tickets#1296] [Help]
Hi,

Is there a way to have custom default settings made for each user when they are created?

Thanks,
Tommy T

**Comment by nblumhardt** [comment]
Hi Tommy,

Under Settings > Users, the "New User Defaults" link provides some options:

![image](https://user-images.githubusercontent.com/342712/129140747-7c8d0ae4-9c0b-4487-8101-5753cfe6f8d2.png)


Is what you're looking for among these?

**Reply by Tthongsy** [reply]
I was looking to see if there was anyway to default a query timeout for new users.
Though I don't think I can find anything in Seq. 

Thanks,
Tommy T

**Reply by nblumhardt** [reply]
Thanks for your reply. No, unfortunately - those defaults can't be set, as things are today, but it's a worthwhile thing for us to consider in a future release 👍
